docs/modules/reference/pages/telemetryd/protocols/bmp.adoc

```diff
@@ -5,7 +5,7 @@ The BGP Monitoring Protocol (BMP) provides a convenient interface for obtaining
 The integration in {page-component-title} lets you use these route views, status updates, and statistics for monitoring and management.
 
 .BMP integration overview
-image:bmp/bmp_integration_overview.png[BMP integration]
+image:bmp/bgp-monitoring.png[BMP integration]
```

Comment:
The diagram needs some updates: We no longer go through Kafka when writing to PostgreSQL and the 'BMP-Postgres' component has been superseded by the `BMP-Persisting-Adapter` adapter.

Reply:
Thanks, @j-white . The whole chapter will need to be revised, as there are a lot of things that have changed. Ronny is currently working on the updates.